### ISSUE TITLE
Add bot to create issues ?

### ISSUE THREAD

**Issue by jswe** [scdotin/test-repo-1#7]

**Comment by chrishna1** [comment]
yeah, that'd be better. if it is configurable(no of issues, pr, rate of issues, pr, etc), we can test multiple cases, which would be difficult otherwise. I'll look into that.
but we'll still need some user account for bio, location, etc.
